hibernate-core/src/test/java/org/hibernate/orm/test/annotations/join/Book.java

```diff
@@ -0,0 +1,45 @@
+package org.hibernate.orm.test.annotations.join;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+
+import java.util.Collection;
+
+@Entity
+@Table(name = "BOOKS")
+public class Book {
+
+	private long bookId;
+
+	private Collection<Page> pages;
+
+	@Id
+	@GeneratedValue
+	@Column(name = "BOOK_ID")
+	public long getBookId() {
+		return bookId;
+	}
+
+	public void setBookId(long userid) {
+		this.bookId = userid;
+	}
+
+	@OneToMany(fetch = FetchType.LAZY)
```

Comment:
OMG man, I wrote _a whole blog post_ explaining quite clearly why you need to use `mappedBy` to achieve high performance here.

I don't know what more I can do to hold your hand on this.

Like .... just _do what it says in the blog_.

Sheeesh.

Reply:
Your original example described in the issue did not use an intermediate join table. It stored the child data directly in the collection table.

So I'm not sure why you're suddenly shifting the goal posts here.

It's by far not the normal thing to map a one-to-many association to a join table.

Anyway, Hibernate has a user forum for asking these kinds of usage questions. Please move this discussion over there.